Sources/BlueSteel/AvroValueConvertible.swift

```diff
@@ -76,13 +76,13 @@ extension Data: AvroValueConvertible {
     }
 }
 
-extension Dictionary: AvroValueConvertible where Key == String, Value == AvroValueConvertible {
+extension Dictionary: AvroValueConvertible where Key == String, Value: AvroValueConvertible {
```

Comment:
I recall also having used this syntax and in an earlier Swift version at least it did not work for me. I suppose the tests pass though?

Reply:
Could be, but in the latest Swift version Xcode throws the error `Referencing instance method 'toAvro()' on 'Array' requires the types 'String' and 'AvroValueConvertible' be equivalent`.
In my opinion, usage conforming `Value` to `AvroValueConvertible` protocol, in this case, is more convenient. What do you think?

Reply:
Yes, tests pass though but probably I should add a few additional tests to fix this behavior and be sure that this extension works properly for Arrays and Dictionaries. I'll do it soon :)